tools/base-language/lib/plugins/event-listeners/SearchMetricListener.ts

```diff
@@ -339,7 +339,7 @@ export class SearchMetricListener extends EventListenerPlugin {
         budgetManager: BudgetManager<E>
       ) => {
         // create a new metric manager for this search subject
-        this.currentNamespace = subject.name;
+        this.currentNamespace = subject.path;
```

Comment:
Can you change this to be relative to the root context.

Reply:
I agree that the namespace should be the path relative to the root context, however, this requires changing the entire framework to update this everywhere. Therefore, that task should probably be a separate PR. If the change is made globally, it will be automatically changed here.

Simply adding another property to fix it temporarily, does not make sense to me

Reply:
Agreed, can you make an issue for future reference?

Reply:
Done https://github.com/syntest-framework/syntest-core/issues/366